src/isIsogram.test.js

```diff
@@ -6,4 +6,20 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('empty string', () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it('should be case-insensitive', () => {
+    expect(isIsogram('isisOGraM')).toBe(false);
+  });
+
+  it('Word without duplicated character', () => {
+    expect(isIsogram('twelve')).toBe(false);
```

Comment:
The test description 'Word without duplicated character' is misleading because the word 'twelve' used in the test has duplicated characters.